src/utils/axios.js

```diff
@@ -0,0 +1,14 @@
+import * as realAxios from 'axios';
+import { BASE_URL, USER_SESSION } from './constants';
+
+const { accessToken } = JSON.parse(
+  localStorage.getItem(USER_SESSION)
+).authResult;
```

Comment:
I think this might fail when a user is initially logged out. `JSON.parse(null)` throws `TypeError: JSON.parse(...) is null`